### PR TITLE
Add karma setup to produce code coverage after test run

### PR DESCRIPTION
**Pull request by peter-mouland**
update read me to include dependency installation
capture browserify config set in package.json to help prevent code duplication

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,8 +33,9 @@ Project structure for new Origami components.
   ```
   git remote add origin https://github.com/Financial-Times/o-your-component.git
   ```
-5. Test and verify: `obt test && obt verify` (and fix the code raising errors)
-6. Commit and push: `git add . && git commit -m "Initial commit" && git push origin master`
+5. install dependencies: `npm i && bower i`
```
Comment (kaelig) [inline README.md:36]: Capital i in "Install" at the beginning of the list item

File: karma.conf.js [hunk 2]
```diff
@@ -65,10 +65,33 @@ module.exports = function(config) {
...
+    var pkg = require('./package.json');
+    karmaConfig.browser = pkg.browser || {};
+    karmaConfig["browserify-shim"] = pkg["browserify-shim"] || {};
+    karmaConfig.browserify = pkg.browserify || {};
+    if (karmaConfig.browserify.transform) {
+        karmaConfig.browserify.transform.push('browserify-istanbul');
+    } else {
+        karmaConfig.browserify.transform = [ 'browserify-istanbul' ];
+    }
+    return config.set(karmaConfig);
```
Comment (kaelig) [inline karma.conf.js:96]: Indentation: tabs are expected in JS files

**Comment by kaelig** [comment]
Thanks Peter!

I agree with everything in there, great stuff.

The process is usually to:
1. try out the new approach (here: code coverage) on one or two modules that have users
2. Learn from that experience and gather feedback from developers
3. Decide if it should be declared a "global best practice"

Are there any existing examples of usage for code coverage in Origami modules?
When we have 1 or 2 we can merge this branch in the boilerplate.

**Comment by kaelig** [comment]
(unless this is a no-brainer and all JS developers want that in now)

/cc @wheresrhys @matthew-andrews @AlbertoElias @kavanagh

**Comment by kavanagh** [comment]
I'd advocate putting this in a "recipes" document rather in the main boilerplate.

@callumlocke has some [good ideas](https://github.com/callumlocke/origami-product-boilerplate/tree/master/docs/recipes) on how to do this in a blueprint/boilerplate project.

File: karma.conf.js [hunk 3]
```diff
@@ -65,10 +65,33 @@ module.exports = function(config) {
...
+            ]
+        }
+	};
+
+    //capture browserify config set in package.json to help prevent code duplication
+    var pkg = require('./package.json');
+    karmaConfig.browser = pkg.browser || {};
+    karmaConfig["browserify-shim"] = pkg["browserify-shim"] || {};
+    karmaConfig.browserify = pkg.browserify || {};
+    if (karmaConfig.browserify.transform) {
```
Comment (wheresrhys) [inline karma.conf.js:91]: the origami build always uses debowerify so you can safely assume browserify.transform exists

**Comment by peter-mouland** [comment]
just doing some spring cleaning and closing old issues - this project looks a little quiet now.

**Pull request closed without merging**